### PR TITLE
tmux: update to 3.5_1

### PR DESCRIPTION
**Pull request by diogom93**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 15.0 24A335 arm64
Command Line Tools 16.0.0.0.1.1724870825

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [x] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL in commit message? 
- [x] checked your Portfile with `port lint --nitpick`?
- [x] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?
- [x] checked that the Portfile's most important [variants](https://trac.macports.org/wiki/Variants) haven't been broken?

### CODE REVIEW

File: sysutils/tmux/Portfile
Comment (tessus) [inline sysutils/tmux/Portfile:12]:
```suggestion
    revision        0
```

File: sysutils/tmux/Portfile
Comment (tessus) [inline sysutils/tmux/Portfile:18]:
```suggestion
    revision        0
```

**Review by tessus — CHANGES_REQUESTED** [review]
The revision must be set to 0.

**Review by tessus — APPROVED** [review]
LGTM.

**Comment by tessus** [comment]
I don't have merge access, thus we'll have to wait for someone to merge this. Thanks for the update.

**Comment by herbygillot** [comment]
I only spotted this after merging, so my fault, but the corresponding `revision` should reset to zero on an update to a port or support's version.

**Comment by herbygillot** [comment]
Ah, I see the revision was reset in a separate commit. That should be happening in the same commit as the update for reference. Thanks.

**Comment by tessus** [comment]
Yep, I usually do squash-and-merge thus I didn't care about the second commit. @diogom93 for refeerence: all PRs should have only one commit (you can always do a squash and force-puah)

**Comment by diogom93** [comment]
Thanks @tessus @herbygillot, first time contributing, I'll keep an eye out next time

**Comment by herbygillot** [comment]
> Thanks @tessus @herbygillot, first time contributing, I'll keep an eye out next time

Understandable - thank you for contributing

**Comment by cjones051073** [comment]
This should not have been merged as is. The first commit comment is meaningless and does not follow our guidelines. It should have been squashed into the second commit. @herbygillot please be more careful in future.

**Comment by herbygillot** [comment]
Yes, my fault for missing that.

**Pull request merged**
